README.md

```diff
@@ -22,9 +22,12 @@ Once you have installed [docker-engine][docker-engine] and
 [Thoughtbot][install-docker]) you can build and run the test suite with:
 
 ```
-./build-and-run-tests.sh
+make -j4
```

Comment:
might be clearer to have it as just `make` and the suggest they use `--jobs 4` as a way to speed up the cloning, could feel a little less magical (or could just be me unfamiliar)